Classes/Issues/Merge/IssueMergeSectionController.swift

```diff
@@ -17,11 +17,18 @@ ListBindingSectionControllerSelectionDelegate {
     private let model: IssueDetailsModel
     private let client: GithubClient
     private let resultID: String?
+    private var mergeCapable = false
```

Comment:
`isMergeCapable` would be better/clearer

Reply:
I was actually thinking about this was gonna ask. isMergeCapable is def more swift style.